src/Views/Cipher/Autokey.tsx

```diff
@@ -7,6 +7,14 @@ class Autokey extends React.Component {
     return (
       <div className="Autokey">
         <div className="Autokey-content">
+          <h2>Autokey Cipher</h2>
+          <p>
+            An autokey cipher is similar to a Vigenere cipher, where the plaintext is used after the key runs out. 
+            This way the repetition of a Vigenere cipher is solved.
```

Comment:
Nit: "is solved" is a bit ambiguous here.  Consider "This addresses the key length repetition issues with the Vigenere cipher."